packages/story-editor/src/components/audioPlayer/index.js

```diff
@@ -96,8 +97,26 @@ function AudioPlayer({ title, src, mimeType }) {
   return (
     <Wrapper>
       {/* eslint-disable-next-line jsx-a11y/media-has-caption, styled-components-a11y/media-has-caption */}
-      <Audio crossOrigin="anonymous" loop ref={playerRef}>
+      <Audio
+        crossOrigin="anonymous"
+        loop
+        ref={playerRef}
+        id={audioId || undefined}
```

Comment:
This is a bit pointless. `audioId` above has an empty string as default value but here you set `undefined` if it's falsey.

Instead you could just remove the default value and do `id={audioId}` here.